R/checkSolver.R

```diff
@@ -28,22 +28,27 @@ checkSolver <- function(solverPath = solverPath, solver = solver,
       }
     }
   }
-  
+
   if(!is(solver, "character")){
     stop("solver should be of type character")
   } else {
     ## checking for solver validity (cplex/cbc/lpSolve)
     valid_solver_list <- c("cplex", "cbc", "lpSolve")
     if (!(solver %in% valid_solver_list)){
-      stop(paste0("Select a valid solver option (", 
+      stop(paste0("Select a valid solver option (",
                   paste(valid_solver_list, collapse=", "), ")"))
     }
   }
-  
+
+  if(solver == "cbc" && mulT){
```

Comment:
make != "cplex"

Reply:
fixed in https://github.com/saezlab/CARNIVAL-Bioconductor-Dev/pull/10/commits/361e7e95a6f42b22c57b3a0b6282237c8d2076c2